domain/src/test/java/org/oppia/android/domain/topic/TopicControllerTest.kt

```diff
@@ -1052,6 +1052,40 @@ class TopicControllerTest {
     assertThat(ongoingTopicList.topicList[0].topicId).isEqualTo(RATIOS_TOPIC_ID)
   }
 
+  @Test
+  fun testOngoingTopicList_finishOneStory_ongoingListIsCorrect() {
+    // Mark entire fractions topic & only 1 chapter in ratios as finished.
```

Comment:
This comment does not make sense. You are not changing anything in `fractions` here.

Reply:
Removed